### PR TITLE
Feature/278 adds onboarding framework

### LINKED ISSUE #278: Add on-boarding framework
We need integrate intro.js or similar framwork to help user understand how Jitsu work. The tour should highlight the following areas:

 - API Keys
 - Sources
 - Destinations

### PR DESCRIPTION
**Pull request by taletski**
closes #278

### CODE REVIEW

File: configurator/frontend/src/lib/components/OnboardingTour/steps/OnboardingTourAddDestination/OnboardingTourAddDestination.module.less [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  margin-bottom: 10px;
+}
+
+.destinationEditorContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  flex: 1 1 100%;
+  width: 100%;
+}
```
Comment (vklimontovich) [inline configurator/frontend/src/lib/components/OnboardingTour/steps/OnboardingTourAddDestination/OnboardingTourAddDestination.module.less:36]: I'd rather use tailwind classes here via `@apply`. For consistency reasons

File: configurator/frontend/src/lib/components/OnboardingTour/steps/OnboardingTourAddDestination/OnboardingTourAddDestination.module.less [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+.paragraph {
+  font-size: @text-font-size;
+  margin-bottom: 20px;
+}
+
+.addDestinationButtonContainer {
+  margin-bottom: 10px;
+}
```
Comment (vklimontovich) [inline configurator/frontend/src/lib/components/OnboardingTour/steps/OnboardingTourAddDestination/OnboardingTourAddDestination.module.less:28]: All `margins-*` instructions would be better done with [Tailwind's m* classes](https://tailwindcss.com/docs/margin)

File: configurator/frontend/src/lib/components/OnboardingTour/steps/OnboardingTourAddDestination/OnboardingTourAddDestination.module.less (hunk 1 above)
Comment (taletski) [reply configurator/frontend/src/lib/components/OnboardingTour/steps/OnboardingTourAddDestination/OnboardingTourAddDestination.module.less:36]: Sure, will do. Thanks for pointing to that!

File: configurator/frontend/src/lib/components/OnboardingTour/steps/OnboardingTourAddDestination/OnboardingTourAddDestination.module.less (hunk 2 above)
Comment (taletski) [reply configurator/frontend/src/lib/components/OnboardingTour/steps/OnboardingTourAddDestination/OnboardingTourAddDestination.module.less:28]: Yep, will stick Tailwind. Thanks!

**Pull request merged**